spec/models/project_spec.rb

```diff
@@ -293,17 +293,39 @@
   end
 
   describe '.goal_between' do
-    let(:start_at) { 100 }
-    let(:ends_at) { 200 }
-    subject { Project.goal_between(start_at, ends_at).order(:id) }
+    let(:start_at)                 { 200 }
+    let(:ends_at)                  { 400 }
+    let!(:all_projects)            { projects_under_start_at + projects_over_ends_at + projects_goal_between }
```

Comment:
It does not matter the order you are declaring the variables in this case. When you call the variable, if it does not exist yet, it is created.
I did this because I think this way is easier to read. Do you agree?